distribution/docker/build.gradle

```diff
@@ -40,40 +48,43 @@ configurations {
   dockerSource
   log4jConfig
   tini
+  repositoryS3Plugin
+  repositoryGcsPlugin
+  repositoryAzurePlugin
+  filebeat
+  metricbeat
 }
 
+String beatsArch = Architecture.current() == Architecture.AARCH64 ? 'arm64' : 'x86_64'
+String tiniArch = Architecture.current() == Architecture.AARCH64 ? 'arm64' : 'amd64'
+
 dependencies {
   aarch64DockerSource project(path: ":distribution:archives:linux-aarch64-tar", configuration: 'default')
   dockerSource project(path: ":distribution:archives:linux-tar", configuration: 'default')
   log4jConfig project(path: ":distribution", configuration: 'log4jConfig')
-  tini 'krallin:tini:0.19.0@tini-amd64'
+  tini "krallin:tini:0.19.0:${tiniArch}"
+  repositoryS3Plugin project(path: ':plugins:repository-s3', configuration: 'zip')
+  repositoryGcsPlugin project(path: ':plugins:repository-gcs', configuration: 'zip')
+  repositoryAzurePlugin project(path: ':plugins:repository-azure', configuration: 'zip')
+  filebeat "beats:filebeat:${VersionProperties.elasticsearch}:${beatsArch}@tar.gz"
```

Comment:
Does this work itself out to a version with a build ID in case of release manager builds? I think we need that to guarantee consistency in the unified release process and make sure that we are picking up the beats from the same build. Also means that ES needs to be built after beats as part of that process. 

Also, someone on the beats team might want to customize this, e.g. check out some ES tag to get a stable/known to be good ES and build the docker images with a custom working version of beats. That's a use case for PRs against beats, but I think it's this code that needs to enable it somehow. Just a taught, it can be done in a follow up PR too.

Reply:
The corresponding release-manager changes fetch the beats artifacts using the build ID, so everything should match up.